python/sdist/amici/pysb_import.py

```diff
@@ -646,17 +704,22 @@ def _add_expression(
             else "normal"
         )
 
-        y = sp.Symbol(f"{name}")
+        y = sp.Symbol(name)
         trafo = noise_distribution_to_observable_transformation(noise_dist)
-        obs = Observable(y, name, sym, transformation=trafo)
-        ode_model.add_component(obs)
-
-        sigma_name, sigma_value = _get_sigma_name_and_value(
-            pysb_model, name, sigmas
+        # note that this is a bit iffy since we are potentially using the same symbolic identifier in expressions (w)
+        # and observables (y). This is not a problem as there currently are no model functions that use both. If this
+        # changes, I would expect symbol redefinition warnings in CPP models and overwriting in JAX models, but as both
```

Comment:
I think these warnings are easily overlooked. Maybe we should turn them into errors. But this will break the import of at least one benchmark collection model that has something named `NULL`. Fine for now.